src/FindAnApprenticeship/SFA.DAS.FindAnApprenticeship/Application/Queries/Apply/GetApplicationSubmitted/GetApplicationSubmittedQueryHandler.cs

```diff
@@ -26,7 +25,14 @@ public class GetApplicationSubmittedQueryHandler : IRequestHandler<GetApplicatio
 
     public async Task<GetApplicationSubmittedQueryResult> Handle(GetApplicationSubmittedQuery request, CancellationToken cancellationToken)
     {
-        var application = await _candidateApiClient.Get<GetApplicationApiResponse>(new GetApplicationApiRequest(request.CandidateId, request.ApplicationId, false));
+        var applicationTask = _candidateApiClient.Get<GetApplicationApiResponse>(new GetApplicationApiRequest(request.CandidateId, request.ApplicationId, false));
+        var aboutYouTask = _candidateApiClient.Get<GetAboutYouItemApiResponse>(new GetAboutYouItemApiRequest(request.CandidateId));
```

Comment:
I _think_ this is probably better than setting the flag on the first request to true and bringing back the whole application 🤔